tool/planet/args.go

```diff
@@ -0,0 +1,82 @@
+/*
+Copyright 2020 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package main
+
+import (
+	"fmt"
+	"net"
+	"strings"
+
+	kv "github.com/gravitational/configure"
+	"github.com/gravitational/trace"
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+func cidrFlag(s kingpin.Settings) *cidr {
```

Comment:
Opted to reimplement the CIDR flag from the [original](https://github.com/gravitational/configure/blob/master/cidr.go#L42) as I needed to have access to the underlying `net.IPNet`.